src/Common/tests/System/Globalization/GlobalizationMode.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization
+{
+    internal sealed partial class GlobalizationMode
+    {
+        internal static bool Invariant { get; } = false;
```

Comment:
I wasn't exactly sure what to do about the tests; we added the `GlobalizationMode.Invariant` conditional to them in Mono, so we can test both modes.

Reply:
@danmosemsft @stephentoub how do you test for such general properties? Do you have any fancy attribute which could run each test or suite in both modes or something like that?

Reply:
corefx\src\System.Globalization\tests\Invariant is the test with Invariant mode. 

I don't think we need to have this new defined class.

Also, please move this PR to coreclr


---
In reply to: [290961607](https://github.com/dotnet/corefx/pull/38281#discussion_r290961607) [](ancestors = 290961607)